Mock_Jetson.py

```diff
@@ -0,0 +1,119 @@
+from Client import *
+import time
+from SensorState import *
+import RandomObjects
+from RandomObjects import RandomObjects
+import GenerateSensorData
+import grid
+from tkinter import *
+import math
+from Consts import *
+from GenerateSensorData import GenerateSensorData
+from EndpointInput import *
+
+class Mock_Jetson:
+    def __init__(self):
+        """
+        """
+        self.grid = grid.Grid(tile_num_height, tile_num_width, tile_size)
+        self.client = Client()
+        self.pos = (0, 0)
+        self.sensor_state = SensorState()
+
+        #Visualization
+        self.master: Tk = Tk()
+        self.canvas: Canvas = None
+        self.heading = 0
```

Comment:
Add a TODO to update the heading in the future

Reply:
resolved